cmd/spiffe-helper/config/config.go

```diff
@@ -15,28 +15,35 @@ import (
 	"github.com/spiffe/spiffe-helper/pkg/sidecar"
 )
 
+const (
+	DaemonModeFlagName = "daemon-mode"
+)
+
```

Comment:
Why the change to this?

Reply:
Because it's used in main.go and config.go at the moment.

Reply:
So you are asking me to put it back and duplicate the const in the config package?

Reply:
Unfortunately yes. I dont like the duplication either but its not related to this changeset.